### ISSUE TITLE
`#include <glslang/SPIRV/SpvTools.h>` fails finding `../glslang/MachineIndependent/localintermediate.h`

### ISSUE THREAD

**Issue by haasn** [KhronosGroup/glslang#2007]
In version 7.13, you moved the header files from `SPIRV/SpvTools.h` to `glslang/SPIRV/SpvTools.h`, which broke several downstream projects. In commit 199d115e, you then fixed this again by installing the files to both directories, with a deprecation notice in the README.

However, even on current master, trying to actually include SpvTools.h from the new path results in this error:

```
In file included from /usr/include/glslang/SPIRV/GlslangToSpv.h:42,
                 from ../src/glsl/glslang.cc:30:
/usr/include/glslang/SPIRV/SpvTools.h:49:10: fatal error: ../glslang/MachineIndependent/localintermediate.h: No such file or directory
   49 | #include "../glslang/MachineIndependent/localintermediate.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

This relative include only makes sense when including from `SPIRV/SpvTools.h`, not from `glslang/SPIRV/SpvTools.h`.

In other words, as of right now, it's actually impossible to include this header successfully in version 7.13 - the only way to build projects against glslang currently is to either downgrade to 7.12 or upgrade to git master and use the old include path.

I highly recommend you make sure the new include path actually *works* before deprecating/removing the old one.

**Comment by johnkslang** [comment]
Agreed.  cc @dj2

**Comment by dj2** [comment]
Going through the headers it looks like GlslangToSpv and SpvTools both have relative include paths to the `../glslang/MachineIndependent`.  Making those just `glslang/MachineIndependent` should work, but will mean folks have to have the include path in their build. Which, I'm assuming they would already ....

**Comment by johnkslang** [comment]
Side note: This is partly because of using originally non-public headers (localIntermediate.h) in what is now a public interface (`SpvTools.h`).

>  it's actually impossible to include this header successfully in version 7.13

I assume you mean for some external ways of building tools from glslang headers.  Of course, it is working fine in the current build for many platforms for building glslangValidator.

I'll fix the relative includes in the headers, and between that and the other open issue on header movement, can close this one.
